api/ServiceWorker.json

```diff
@@ -35,6 +35,41 @@
           "deprecated": false
         }
       },
+      "ecmascript_modules": {
+        "__compat": {
+          "description": "Support for ECMAScript modules",
+          "support": {
+            "chrome": {
+              "version_added": "91"
```

Comment:
Added in FF91 but only does `import` because that is what spec requires - https://chromestatus.com/feature/4609574738853888